datastructures/Board.cpp

```diff
@@ -142,4 +143,10 @@ void Board::printBoard() {
     }
 }
 
+Piece* Board::getPiece(int row, int col) {
```

Comment:
Doesn't this encourage not using the Position datastructure?

Reply:
Yeah, don't think I use it anyways, will remove it.